### PR TITLE
started clean branch for repeatmodeler recipe

### PR DESCRIPTION
**Pull request by aymanm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/recon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+about:
+  home: http://eddylab.org/software/recon/
+  license: 'GNU General Public License'
+  summary: 'Proper identification of repetitive sequences is an essential step in genome analysis.'
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/recon/meta.yaml]: new recipe should start with 0

File: recipes/repeatscout/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+about:
+  home: http://repeatscout.bioprojects.org
+  license: 'GNU General Public License'
+  summary: 'De novo identification of repeat families in large genomes.'
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/repeatscout/meta.yaml]: 0

File: recipes/repeatmodeler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+about:
+  home: 'http://www.repeatmasker.org/RepeatModeler.html'
+  license: 'Open Source License v2.1'
```
Comment (bgruening) [inline recipes/repeatmodeler/meta.yaml]: is this GPL?

File: recipes/repeatmodeler/build.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+RM_PROGRAMS="RepeatModeler $RM_OTHER_PROGRAMS"
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${RM_DIR}
+cp -r * ${RM_DIR}
+for name in ${RM_PROGRAMS} ; do 
+  perl -i -0pe 's/^#\\!.*perl.*/#\\!\/usr\/bin\/perl/g;' ${RM_DIR}/${name}
+done
+
+cp ${RECIPE_DIR}/RepModelConfig.pm ${RM_DIR}/RepModelConfig.pm
```
Comment (bgruening) [inline recipes/repeatmodeler/build.sh]: can you add a comment why this is needed?

File: recipes/repeatscout/meta.yaml (hunk 2 above)
Comment (aymanm) [inline recipes/repeatscout/meta.yaml]: fixed

File: recipes/recon/meta.yaml (hunk 1 above)
Comment (aymanm) [inline recipes/recon/meta.yaml]: fixed

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**